File: src/main/java/com/example/workflowmassive/WorkflowMassiveApplication.java
package com.example.workflowmassive;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.kafka.annotation.EnableKafka;

@SpringBootApplication
@EnableKafka
public class WorkflowMassiveApplication {

    public static void main(String[] args) {
        SpringApplication.run(WorkflowMassiveApplication.class, args);
    }

}

File: src/main/java/com/example/workflowmassive/WorkflowMassiveApplication.kt
package com.example.workflowmassive;

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.kafka.annotation.EnableKafka

@SpringBootApplication
@EnableKafka
class WorkflowMassiveApplication

fun main(args: Array<String>) {
    runApplication<WorkflowMassiveApplication>(*args)
}

